src/Worker.Extensions.DurableTask/FunctionsDurableTaskClient.cs

```diff
@@ -17,15 +17,16 @@ internal sealed class FunctionsDurableTaskClient : DurableTaskClient
 {
     private readonly DurableTaskClient inner;
 
-    public FunctionsDurableTaskClient(DurableTaskClient inner, string? queryString)
+    public FunctionsDurableTaskClient(DurableTaskClient inner, string? queryString, string? httpBaseUrl)
         : base(inner.Name)
     {
         this.inner = inner;
         this.QueryString = queryString;
+        this.HttpBaseUrl = httpBaseUrl;
     }
 
     public string? QueryString { get; }
-
+    public string? HttpBaseUrl { get; }
```

Comment:
Is it allowed for this string to ever be null? If not, I'd recommend removing the `?` from the `string` type.

Reply:
If  the `DurableTaskClient` isn’t an instance of `FunctionsDurableTaskClient`, then HttpBaseUrl could be null, as it’s only an attribute of `FunctionsDurableTaskClient`. I’m not entirely sure in what scenario the DurableTaskClient wouldn’t be a FunctionsDurableTaskClient—perhaps it could happen if bindings aren’t being used. That’s why I added an exception for cases where the base URL creation fails.